bindings/pyroot_experimental/PyROOT/test/tcomplex_radd_rsub.py

```diff
@@ -0,0 +1,43 @@
+import unittest
+import ROOT
+from ROOT import TComplex
+
+class TestTComplexMethods(unittest.TestCase):
+        """
+        Test for the operators of TComplex:
+        __add__, __sub__, __radd__, __rsub__.
+        """
+
+        c1 = TComplex(4.,0)
+        c2 = TComplex(1.,1)
+
+        d = 2.
+
+        # check if both __add__ and __radd__ give the expected result with complex + double and double + complex and if __radd__ is equal to __add__ both for complex + complex and complex + double (real components)
+        def test_radd(self):
+                self.assertEqual((self.c1 + self.d).Re(), 6.0)
+                self.assertEqual((self.d + self.c1).Re(), 6.0)
+                self.assertEqual((self.c1 + self.c2).Re(), (self.c2 + self.c1).Re())
+                self.assertEqual((self.c1 + self.d).Re(), (self.d + self.c1).Re())
+
+        # check if both __sub__ and __rsub__ give the expected result with complex - double and double - complex and if __rsub is equal to -(__sub__) both for complex - complex and complex - double (real components)
+        def test_rsub(self):
+                self.assertEqual((self.c1 - self.d).Re(), 2.0)
+                self.assertEqual((self.d - self.c1).Re(), -2.0)
+                self.assertEqual((self.c1 - self.c2).Re(), -((self.c2 - self.c1).Re()))
+                self.assertEqual((self.c1 - self.d).Re(), -((self.d - self.c1).Re()))
+
+        # check if both __mul__ and __rmul__ give the expected result with complex * double and double * complex and if __radd__ is equal to __add__ both for complex * complex and complex * double (real components)
+        def test_rmul(self):
+                self.assertEqual((self.c1 * self.d).Re(), 8.0)
+                self.assertEqual((self.d * self.c1).Re(), 8.0)
+                self.assertEqual((self.c1 * self.c2).Re(), (self.c2 * self.c1).Re())
+                self.assertEqual((self.c1 * self.d).Re(), (self.d * self.c1).Re())
+
+        # check if both __truediv__ and __rtruediv__ give the expected result with complex / double and double / complex (real components)
+        def test_rmul(self):
```

Comment:
Should be `test_rtruediv`.

Also, is there a reason why you pythonised `__rtruediv__` and not `__rdiv__`?